profiles/killbill/src/test/java/org/killbill/billing/server/log/obfuscators/TestPatternObfuscator.java

```diff
@@ -17,16 +17,112 @@
 
 package org.killbill.billing.server.log.obfuscators;
 
-import org.killbill.billing.server.log.ServerTestSuiteNoDB;
+import java.util.Collection;
+import java.util.List;
+
 import org.mockito.Mockito;
 import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
 
 import ch.qos.logback.classic.spi.ILoggingEvent;
 
-public class TestPatternObfuscator extends ServerTestSuiteNoDB {
+public class TestPatternObfuscator {
+
+    private PatternObfuscator obfuscator;
+
+    @BeforeClass(groups = "fast")
```

Comment:
Should we have an `@AfterClass` to remove the system properties for cleanliness?

Reply:
Not required now as I've removed the `@BeforeClass` and added an `@AfterMethod` method that clears the system properties.